widget/window_switcher.lua

```diff
@@ -362,20 +343,19 @@ local enable = function(opts)
     }
 
     window_switcher_box:connect_signal("property::width", function()
-        if window_switcher_box.visible and get_num_clients() == 0 then
+        if window_switcher_box.visible and #awful.screen.focused().selected_tag:clients() == 0 then
```

Comment:
stupid question but, what does the `#` prefixed to the function is supposed to do?
Haven't seen that syntax oftenly in lua.